src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -41,29 +41,32 @@
 import org.xembly.Directives;
 
 /**
- * Starts Amazon EC2 instance.
+ * Starts Amazon ECS instance, that running Docker + SSHd on ECS. Instance
+ * to be configured in privileged mode. Configuration needs to be stored in S3.
```

Comment:
@xupyprmv this is Javadoc, so it should tell what the class does, not what needs to be done.